roles/gateway/README.rst

```diff
@@ -58,6 +61,63 @@ A list of users that don't have their ssh pubkey added to the ``teuthology_user`
     openvpn_users:
       - ovpn: user@host etc...
 
+The following vars are used to populate ``/etc/resolv.conf``.  Defined in the
+secrets repo::
+
+    gw_resolv_search: []
+    # Example: gw_resolv_search: "front.example.com"
+
+    gw_resolv_ns: []
+    # Example:
+    gw_resolv_ns:
+      - 1.2.3.4
+      - 8.8.8.8
+
+The ``gw_networks`` var assumes you have individual "physical" NICs for each
+VLAN in your lab.  The subelements ``peerdns`` and ``dns{1,2}`` are optional.
+``dns1`` and ``dns2`` should be defined under a single NIC and ``peerdns``
+should be set to ``"yes"``.  Defined in the
```

Comment:
PEERDNS, in conjunction with DNS{1,2}, adds the nameservers defined by DNS{1,2} to /etc/resolv.conf.  It should only bet set on one NIC.  If set to "yes" on multiple NICs, the last NIC that gets brought up will clobber the first nameservers added.